recipes/tad-libcint/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "tad-libcint" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tad_libcint-{{ version }}.tar.gz
+  sha256: c47332ec4c1b8a177b2d3c8f093a3ae04a507719efda5feb6c91991a23933444
+
+build:
+  skip: true  # [py>=313 or not linux]
```

Don't skip py313 yet or otherwise the migration might not come around later
```suggestion
  skip: true  # [not linux]
```